oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreServiceConfigurationTest.java

```diff
@@ -92,6 +94,14 @@ public void presetMongoURI() throws Exception {
         assertEquals(uri, config.mongouri());
     }
 
+    @Test
+    public void throttleEnabled() throws Exception {
+        boolean throttleDocStore = false;
```

Comment:
This is the default. Shouldn't we test the opposite?

Reply:
Yes, it makes sense to test the opposite of default, will update the test case.